Fable.Remoting.IntegrationTests/Shared/SharedTypes.fs

```diff
@@ -58,7 +58,7 @@ type SomeEnum =
     | Val1 = 1
     | Val2 = 2
 #if FABLE_COMPILER
-[<StringEnum>]
+[<StringEnum(CaseRules.None)>]
 #endif
```

Comment:
Hm, why is this Fable only? If the server doesn't know it's dealing with a string enum, it will use a number.